portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019 Google LLC
```

Comment:
I see this file used to be DataServlet.java.

Instead of removing the file and adding a new one, you can rename it in git using `git mv`. This keeps the rename history in git so the diff isn't so huge.

Reply:
Ahh I see, ok, good to know.